README.md

```diff
@@ -42,23 +30,279 @@ Express has extensive support for template rendering and can load and leverage H
 - **helpers** (functions used in templates to manipulate data) to ultimately render **views**.
 
 ## Dive into coding
-**Note for mentor: Please use the additional readme ([mentors-notes.md](./mentors-notes.md)) and follow step by step instructions for the code-along workshop.**
+
+**The Task**
+
+The task folder in this repo holds an existing site, including multiple html files. Many of these html files include duplicated code. We are going to use the power of handlebars templates, partials and helpers to remove this duplication, dramatically reducing the number of files and amount of code.
+
+### Set Up
 
 1. Clone this repo
 1. `cd task/`
 1. Run `npm install` to install all the Node dependencies
 1. Review the existing folder structure
 
-  Notice that in order to serve individual fruit pages we had to create a number of static `.html` pages.
+  Notice that in order to serve individual fruit pages we have many static `.html` pages.
 
   <img src="./images-readme/folder-structure-fruits-html.png" width="200px" height="auto"/>
 
-  Notice also that we have two `.html` files to handle errors.
+  Notice also that we have two html files to handle errors.
+
+  Handlebars will help us to avoid duplication by creating template views. In fact Handlebars will allow us to delete all those html files.
+
+### Install handlebars
+
+ 1. In the tasks directory run the follwing code:
+
+    ```bash
+    npm i --save express-handlebars
```

Comment:
Unusual to have shorthand for `i` but include `--save`. I would use either
- `npm install --save`
- `npm i`
But that's more of a personal stylistic thing

Reply:
I've made changes in response to all your comments above, see commits below.